### ISSUE TITLE
Feature: Click to search by tag

### ISSUE THREAD

**Issue by dburka** [resolvetosavelives/healthicons#87]
We should make it possible to click on a tag and see all of the icons that match that tag.

Stretch goal: We could show 'related' icons in the modal dialog. So when you click on the "Farmer" icon, we show all other "agriculture" or "worker" icons.

**Comment by Ishita-2611** [comment]
Hey, I would love to work on this issue , can you assign me this .

**Comment by dburka** [comment]
Great @Ishita-2611 , I'm assigning it to you. Thanks for your patience.

**Comment by dburka** [comment]
Yes, this feature is awesome. Great work. It's live on the website, so I'll close the issue here.
